src/scripts/main.js

```diff
@@ -1,3 +1,68 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('tbody');
+
+const buttonAddRow = document.getElementsByClassName('append-row')[0];
+const buttonRemoveRow = document.getElementsByClassName('remove-row')[0];
+const buttonAddCol = document.getElementsByClassName('append-column')[0];
+const buttonRemoveCol = document.getElementsByClassName('remove-column')[0];
+
+const minRowsNumber = 2;
+const maxRowsNumber = 10;
+
+buttonAddRow.addEventListener('click', e => {
```

Comment:
actually you don't need to get 'event' as a parameter, because you are not using it, so write just `() => {}`

Reply:
ok! done